### PR TITLE
Update specialisms

### PR DESCRIPTION
**Pull request by kingkorf**
See https://www.bigregister.nl/doc/pdf/BIG%20register%20services%202.0_43640.pdf

### CODE REVIEW

File: src/Wb/BigRegister/SoapClient/ResponseParser.php [hunk 1]
```diff
@@ -144,65 +145,61 @@ protected function getProfession($code)
...
+            61 => 'Klinische psychologie (klinisch psycholoog)',
+            62 => 'Interne geneeskunde-allergologie',
+            63 => 'Klinische neuropsychologie',
+            65 => 'Verpl. spec. prev. zorg bij som. aandoeningen',
+            66 => 'Verpl. spec. acute zorg bij som. aandoeningen',
+            67 => 'Verpl. spec. intensieve zorg bij som. aandoeningen',
+            68 => 'Verpl. spec. chronische zorg bij som. aandoeningen',
+            69 => 'Verpl. spec. geestelijke gezondheidszorg',
+            70 => 'Jeugdgezondheidszorg (Profiel KNMG Jeugdarts)',
+            71 => 'Spoedeisendehulp  (Profiel SEH Arts KNMG)',
```
Comment (eriwin) [inline src/Wb/BigRegister/SoapClient/ResponseParser.php]: Can you update this to `Spoedeisendehulp (Profiel SEH Arts KNMG)'` with one space

File: src/Wb/BigRegister/SoapClient/ResponseParser.php [hunk 2]
```diff
@@ -144,65 +145,61 @@ protected function getProfession($code)
...
+            31 => 'Oogheelkunde (oogarts)',
+            32 => 'Orthopedie (orthopeed)',
+            33 => 'Pathologie (patholoog)',
+            34 => 'Plastische chirurgie (plastisch chirurg)',
+            35 => 'Psychiatrie (psychiater)',
+            39 => 'Radiologie (radioloog)',
+            40 => 'Radiotherapie (radiotherapeut)',
+            41 => 'Reumatologie (reumatoloog)',
+            42 => 'Revalidatiegeneeskunde (revalidatiearts)',
+            43 => 'Maatschappij en gezondheid  (beëindigd per 01-01-2007)',
```
Comment (eriwin) [inline src/Wb/BigRegister/SoapClient/ResponseParser.php]: Can you update this to `Maatschappij en gezondheid  (beëindigd per 01-01-2007)` with one space

File: src/Wb/BigRegister/SoapClient/ResponseParser.php [hunk 3]
```diff
@@ -144,65 +145,61 @@ protected function getProfession($code)
...
+            63 => 'Klinische neuropsychologie',
+            65 => 'Verpl. spec. prev. zorg bij som. aandoeningen',
+            66 => 'Verpl. spec. acute zorg bij som. aandoeningen',
+            67 => 'Verpl. spec. intensieve zorg bij som. aandoeningen',
+            68 => 'Verpl. spec. chronische zorg bij som. aandoeningen',
+            69 => 'Verpl. spec. geestelijke gezondheidszorg',
+            70 => 'Jeugdgezondheidszorg (Profiel KNMG Jeugdarts)',
+            71 => 'Spoedeisendehulp  (Profiel SEH Arts KNMG)',
+            74 => 'Sportgeneeskunde',
+            75 => 'Openbaar apotheker',
```
Comment (eriwin) [inline src/Wb/BigRegister/SoapClient/ResponseParser.php]: Can you rename this one to `Openbare Farmacie`. The SOAP response of http://webservices.cibg.nl/Ribiz/OpenbaarV4.asmx?op=GetRibizReferenceData returns it that way.

**Comment by eriwin** [comment]
I checked the SOAP response of http://webservices.cibg.nl/Ribiz/OpenbaarV4.asmx?op=GetRibizReferenceData and found a small difference. Can you update `Openbaar apotheker` too?

**Comment by kingkorf** [comment]
There we go! 👍

**Comment by eriwin** [comment]
@kingkorf Thank you very much 👍

**Comment by kingkorf** [comment]
Hi Erik, can you do a patch release (2.0.1?) so I can update my composer.json?

**Comment by eriwin** [comment]
@kingkorf Yes. The 2.0.1 release is added.

**Comment by kingkorf** [comment]
Thanks!

**Pull request merged**